src/transaction/log_manager.c

```diff
@@ -170,6 +170,33 @@ struct archive_log_header_scan_context
   LOG_ARV_HEADER header;
 };
 
+/*It will be moved to the CDC module and will define overflow.h in that module  */
+
+struct overflow_first_part
```

Comment:
It looks good to me if you move this structure and typedef to the CDC header file. I mean I wish you make a CDC header(s) and use it for declarations.